### PR TITLE
Feature/localization

### PR DESCRIPTION
**Pull request by Wuyika**
- added assets for app logo
- done splash screen UI
- added translation files and configurations for localization
- done with onboarding screen
- added static local homepage to move to after onboarding for now

## Sreens
| Row A | Row B | Row C |
| -------- | ---------- | --------- |
| image | image | Add 3rd screen here |
| Repeat pattern here | Continue repeating pattern | yet another screen |

### CODE REVIEW

File: assets/translations/en.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "onboard_desc_1": "Trakli focuses on the basics and gives just what you need to see where you money is coming from or going to.",
+  "onboard_title_2": "Automated",
+  "onboard_desc_2": "Automated",
+  "onboard_title_3": "Free and open source",
+  "onboard_desc_3": "It does not cost anything to use Trakli",
+  "skip": "Skip",
+  "login": "Login",
+  "login_google": "Login with Google",
+  "login_Apple": "Login with Apple"
+}
```
Comment (Fenn-CS) [inline assets/translations/en.json:15]: Thank you! Insert line breaks.

Comment (Wuyika) [reply assets/translations/en.json]: Alright added line breaks

**Comment by Fenn-CS** [comment]
@Wuyika Take note that the logo has been updated.

**Comment by Wuyika** [comment]
> @Wuyika Take note that the logo has been updated.

Updated logo SVGs

**Pull request merged**